src/App.tsx

```diff
@@ -1,26 +1,30 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import logo from './logo.svg'
+import './App.css'
+import axios from 'axios'
+
+const backendAddr = 'http://127.0.0.1:3000/api/v1/ping'
+
+function onClick() {
```

Comment:
I would rather use name like handleClick(), to avoid naming function same way as built in event handler.